src/sections/Meshery/Meshery-platforms/index.js

```diff
@@ -77,8 +76,8 @@ const supported_platforms = [
       <>
         <h2>Helm Chart</h2>
         <p>Install on Kubernetes using Helm:</p>
-        <Code codeString={dedent`helm repo add meshery https://meshery.io/charts/
-        helm install my-meshery meshery/meshery --version 2.1.2`}
+        <Code codeString={dedent`kubectl create namespace meshery
+        helm install meshery --namespace meshery install/kubernetes/helm/meshery`}
```

Comment:
Thank you, @debjani1413! Its turns out that the commands on meshery.io needed to be updated, too! I'm sorry for the run around here. This PR has all of the latest and correct commands. Will you please consider refreshing using this set? https://github.com/meshery/meshery.io/pull/538